specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/databases.json

```diff
@@ -968,6 +974,10 @@
           "description": "The date when database was resumed by user action or database login (ISO8601 format). Null if the database is paused.",
           "type": "string",
           "readOnly": true
+        },
+        "maintenanceConfigurationId": {
```

Comment:
>maintenanceConfigurationId [](start = 9, length = 26)

You are adding a new property to an existing API version. Is it possible for existing clients that are not aware of `maintenanceConfigurationId` properties to cause data loss when interacting with an updated backend?

Reply:
On the backend we are going to do nothing if 'null' or nothing is specified. So no, data loss is not going to be possible.

Reply:
Sounds good.